src/new/sendTransaction.ts

```diff
@@ -262,23 +262,27 @@ export default class SendTransaction extends EventEmitter {
    * Prepare transaction data from inputs and outputs
    * Fill the inputs if needed, create output change if needed and sign inputs
    *
+   * @param {string | null} Pin to use in this method (overwrites this.pin)
```

Comment:
chore(minor): First word after the type is the parameter name. After it is the description
```suggestion
   * @param {string | null} pin Pin to use in this method (overwrites this.pin)
```

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/948/commits/29370101b20973e0bc4287dc030f7ca661d6f0d6